adaptive_navigation/example/lib/custom_scaffold.dart

```diff
@@ -27,6 +29,8 @@ class _CustomScaffoldDemoState extends State<CustomScaffoldDemo> {
           return NavigationType.bottom;
         }
       },
+      fabInRail: _fabInRail,
+      overflowDestinationAddedToMenu: _overflowDestinationAddedToMenu,
```

Comment:
The overflow destinations are always added to menu, so this should be something like "includeBaseDestinationsInMenu"

Reply:
Done